SpiffWorkflow/dmn/serializer/task_spec_converters.py

```diff
@@ -91,6 +92,7 @@ def output_entry_from_dict(self, dct, outputs):
     def rule_from_dict(self, dct, inputs, outputs):
         rule = Rule(dct['id'])
         rule.description = dct['description']
+        rule.row_number = dct['row_number']
```

Comment:
This should be changed to dct.get('row_number').  It might not be present if the workflow was serialized before this change goes in.